graphql-protocol/src/main/java/com/linecorp/armeria/internal/server/graphql/protocol/GraphqlUtil.java

```diff
@@ -48,6 +49,10 @@ public static MediaType produceType(RequestHeaders headers) {
                 contentType != null && contentType.is(MediaType.GRAPHQL)) {
             return MediaType.GRAPHQL_JSON;
         }
+        if (HttpMethod.POST == headers.method() &&
+                contentType != null && contentType.is(MediaType.MULTIPART_FORM_DATA)) {
+            return MediaType.GRAPHQL_JSON;
+        }
```

Comment:
The branch above this `if` also checks HTTP method. How about optimizing a little bit? i.e.

```java
if (headers.method() == HttpMethod.POST && contentType != null) {
    if (contentType.is(MediaType.GRAPHQL) {
        return MediaType.GRAPHQL_JSON;
    } else if (contentType.is(MediaType.MULTIPART_FORM_DATA) {
        return MediaType.GRAPHQL_JSON;
    }
}
```